src/transformers/image_processing_utils.py

```diff
@@ -13,17 +13,31 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from typing import Dict, Iterable, Optional, Union
+import copy
```

Comment:
We have 8 `image_processing_utils.FeatureExtractionMixin` in  this file, which should be changed, right?

Reply:
Right. Updated now :)